helms/odahu-flow-packagers/values.yaml

```diff
@@ -24,3 +24,6 @@ packagers:
     # This variable can be uncommented to setup custom image name for REST packager
     # Type: string
     # image: custom-image:1.0
+
+  cli:
+    enabled: true
```

Comment:
Please, repeat and add documentation and the image parameter like in the rest option.